docs/README.md

```diff
@@ -0,0 +1,32 @@
+# Mintlify Starter Kit
+
+Click on `Use this template` to copy the Mintlify starter kit. The starter kit contains examples including
+
+- Guide pages
+- Navigation
+- Customizations
+- API Reference pages
+- Use of popular components
+
+### Development
+
+Install the [Mintlify CLI](https://www.npmjs.com/package/mintlify) to preview the documentation changes locally. To install, use the following command
+
+```
+npm i -g mintlify
+```
+
+Run the following command at the root of your documentation (where mint.json is)
+
+```
+mintlify dev
```

Comment:
Could you also replace the docs-related commands in the Makefile to use mintlify instead? Would be great to install/verify the installation through npm as well, maybe an update to the dev requirements install command.

Reply:
Added a `make docs` command that cd's into the docs repo and runs `npx mintlify dev` so even if folks don't have mintlify downloaded locally it should run & serve the docs. node would be required for npm / npx though so not entirely sure where would be a good place to mention that in the README